server/controllers/system/add-department.php

```diff
@@ -29,20 +28,15 @@ class AddDepartmentController extends Controller {
     public function validations() {
         return [
             'permission' => 'staff_3',
-            'requestData' => [
-                'name' => [
-                    'validation' => DataValidator::alnum(),
-                    'error' => ERRORS::INVALID_NAME
-                ]
-            ]
+            'requestData' => []
         ];
     }
 
     public function handler() {
         $name = Controller::request('name');
```

Comment:
We shouldn't only limit to departments, can you check if ticket title have this issue too?

Reply:
there isn't problem with that path , because it hasn't alphanumeric validation